src/main/java/com/shopify/model/ShopifyPage.java

```diff
@@ -0,0 +1,42 @@
+package com.shopify.model;
+
+import java.util.ArrayList;
+
+public class ShopifyPage<T> extends ArrayList<T> {
+
+	private static final long serialVersionUID = 7202410951814178409L;
+
+	private String nextPageInfo;
+	private String previousPageInfo;
+
+	public String getNextPageInfo() {
+		return nextPageInfo;
+	}
+
+	public void setNextPageInfo(final String nextPageInfo) {
+		this.nextPageInfo = nextPageInfo;
+	}
+
+	public String getPreviousPageInfo() {
+		return previousPageInfo;
+	}
+
+	public void setPreviousPageInfo(final String previousPageInfo) {
+		this.previousPageInfo = previousPageInfo;
+	}
+
+	@Override
+	public int hashCode() {

```

Comment:
Please add some assertions either in ShopifySDKTest or in a new test class around this hashCode